test/StudentTest.java

```diff
@@ -28,4 +31,21 @@ public void shouldThrowExceptionWhenSettingNameToNull() {
             //Success
         }
     }
+
+    @Test
+    public void shouldReturnTheIdThatWasSet(){
+        classUnderTest.setId("123456");
+        String testId=classUnderTest.getId();
+        assertEquals("student id if not what expected","123456",testId);
+    }
+
+    @Test
+    public void shouldThrowExceptionWhenSettingIdToNull() {
+        try {
+            classUnderTest.setId(null);
+            fail();
```

Comment:
another option is to use the following annotation (expected=<Exception>.class) right next to the Test annotation. For example:

@Test(expected=RunTimeException.class)

Than the test will look as followed:

@Test(expected=RunTimeException.class)
public void shouldThrowExceptionWhenSettingIdToNull() {
      classUnderTest.setId(null);
}